lib/config.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+
+var config = require('bedrock').config;
+var path = require('path');
+require('bedrock-validation');
+
+config['messages-client-http'] = {};
+config['messages-client-http'].basePath = '/messages-client';
```

Comment:
We've recently adopted a `.routes = {}` approach and put `basePath` inside `routes` along with anything else.